src/components/features/ProgressBar.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="progress progress-striped">
+    <div class="progress__bar"></div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "ProgressBar",
+  props: ["startPercent", "config"],
+  data() {
+    return {
+      progressBar: null,
+      progressBarContainer: null
+    };
+  },
+
+  created() {
+    this.initializeProgressBar();
+  },
+
+  computed: {
+  },
+
+  methods: {
+    initializeProgressBar(){
+      setTimeout(() => {
+        this.progressBarContainer = document.querySelector(".progress")
+        this.progressBar = document.querySelector(".progress__bar")
+        this.setConfigProperties(this.config['details'])
+        this.progressTo(this.startPercent)
+      }, 200)
+    },
+
+    setConfigProperties(config){
```

Comment:
Rename to `setCSSProperties`, or if you expect that other configs will be set, create a method called `setCSSProperties` and call it from here. Separating CSS parts from code parts is super important.

You should also add a comment in the CSS itself, saying that these properties are set in the code